### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by kerneljay**
Tweaks include;
- Force first person (config option)
- Change Z via up/down arrow

### CODE REVIEW

File: client/cl_main.lua [hunk 1]
```diff
@@ -90,14 +103,14 @@ local function PlacingThread(animData)
                     heading = heading - 5
                     if heading < 0 then heading = 360.0 end
                 end
-                if IsControlJustPressed(0, 38) then
-                    if #(GetEntityCoords(PlayerPedId()) - currentCoords) < 5.0 then
+                if IsControlJustReleased(0, 38) then
+                    if #(GetEntityCoords(PlayerPedId()) - currentCoords) < 35.0 then
```
Comment (pushkart2) [inline client/cl_main.lua:107]: this is just exploitable. This means a player can push the ped 35 units away and still just walk all the way there.

File: client/cl_main.lua [hunk 2]
```diff
@@ -134,16 +146,16 @@ local function RotationToDirection(rotation)
...
 	local cameraCoord = GetGameplayCamCoord()
 	local direction = RotationToDirection(cameraRotation)
 	local destination =
 	{
-		x = cameraCoord.x + direction.x * 10.0,
-		y = cameraCoord.y + direction.y * 10.0,
-		z = cameraCoord.z + direction.z * 10.0
+		x = cameraCoord.x + direction.x * 1000.0,
+		y = cameraCoord.y + direction.y * 1000.0,
+		z = cameraCoord.z + direction.z * 1000.0
```
Comment (pushkart2) [inline client/cl_main.lua:157]: you dont need it to be 1000, you can get away with 20 or something once you reduce the distance above

Comment (kerneljay) [reply client/cl_main.lua:157]: my bad, i forgot to revert these. testing

File: client/cl_main.lua (hunk 1 above)
Comment (kerneljay) [reply client/cl_main.lua:107]: my bad, i forgot to revert these. testing

**Comment by pushkart2** [comment]
Taken most of the necessary changes from here and put it in the commit https://github.com/snipe-scripts/snipe-sitting/commit/572254cce45405fc2c101f24e9d1fe54b7c84d98

**Pull request closed without merging**